google/cloud/pubsublite/internal/resumable_async_streaming_read_write_rpc.h

```diff
@@ -59,6 +59,34 @@ using RetryPolicyFactory = std::function<std::unique_ptr<RetryPolicy>()>;
 /**
  * The purpose of this class is to encapsulate the retry logic of a generic
  * bidirectional asynchronous RPC stream.
+ *
+ * Example (unrealistic) usage:
+ * ```
+ * std::string ExampleReadUsage() {
+ *   ResumableAsyncStreamingReadWriteRpc<Request, Response> resumable_stream;
+ *   auto start = resumable_stream.Start();
+ *   size_t num_attempts = 0;
+ *   while (num_attempts < 3) {
```

Comment:
yea this has to do with the 'unrealistic' part
I just wanted a loop to show how you can automatically retry your `Read` call. `num_attempts` is intended as just a counter variable without any legitimate usage beyond that despite its name

yup it would be the same for `Write`